packages/local_storage/lib/src/local_storage.dart

```diff
@@ -26,51 +21,55 @@ class LocalStorage {
       version: 1,
       onCreate: (db, version) {
         db.execute('''
-          CREATE TABLE $_tasksTableName(
-            $_tasksIdColumnName INTEGER PRIMARY KEY,
-            $_tasksContentColumnName TEXT NOT NULL,
-            $_tasksStatusColumnName INTEGER NOT NULL
+          CREATE TABLE tasks(
+            id INTEGER PRIMARY KEY,
+            conctent TEXT NOT NULL,
+            status INTEGER NOT NULL
```

Comment:
I don't know what to do with this peace of code. I don't like you create tasks table in LocalStorage. It's important to understand that LocalStorage is something abstract - it should not mention task anywhere. You should use LocalStorage in TasksRepository and there adjust to your needs - give a database name and properties, names of columns. I'm not sure how to achieve it with this particular package. Wait- look at the example before change - in lines 29-32 you use variables. That means you can try to write a method createDatabase which creates database based on given parameters and returns database instance. Let's create such a method. Remember: every tasks related settings should be made in TasksRepository, but you have to make LocalStorage flexible enough to make such a settings there.